lib/cli.rb

```diff
@@ -0,0 +1,31 @@
+class CLI
```

Comment:
Great job introducing CLI class! and I like the `put_string` method, because that's exactly what CLI is doing.

I'm hesitant about put_board however, because it seems like it is doing the same thing, outputting a string, just string is different

Reply:
I changed the method `put_board` to `generate_board_str`. It now returns a string, but does not print it at all.

Do you think this would be a better approach? I think this better follows the SRP.

Reply:
yes and no :)
yes - now method name actually describes what is does, it generates a string, that's good.
no - new questions arise now:
1 - generating a string, is it something `CLI` should be doing? 
2 - I can't see if the board is actually being printed anywhere

Reply:
1. I think generating the string should go in `Board` class. But, the problem I have with that approach is that generating a string board seems specific to CLI. For example, a website wouldn't need a string version of the board. 
2. I don't think I quite understand your second comment. I was thinking it would be printed like this:
```
board_state = board.board
board_string = board.generate_board_str(board_state)
cli.put_string board_string
```

Reply:
I have an idea of what I could do. For comment #2 I think the print, I think I could put the printing in `TicTacToe` class. I will leave another comment here when I push the changes.

Reply:
If you have trouble finding a class whose responsibility it is to generate the board string, maybe you need a new class?

Reply:
I talked a little bit to Olga about this, and I created a UserInterface class that bridges the gap between the CLI and the game. 

I thought that CLI should still convert the board to text, because that is a CLI specific action. For example, a web app would not need to display it as a string.